### PR TITLE
Homework_3

### PR DESCRIPTION
**Pull request by IrinaBondarenko**

### CODE REVIEW

File: homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+    return NewArray;
+}
+console.log(array(50, 5));
+console.log(array(32, 4));
+const zArray = array(32, 4);
+const xArray = [];
+for (let i = 0; i < zArray.length; i++) {
+    xArray[i] = degree(zArray[i]);
+}
```
Comment (markoffden) [inline homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js]: please use `map()` method here

File: homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+console.log(xArray);
+let sum = 0;
+for (let i = 0; i < xArray.length; i++) {
+    sum += xArray[i];
+}
+console.log(sum);
```
Comment (markoffden) [inline homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js]: please use `map()` method here

File: homeworks/irina.vlasova_IrinaBondarenko/homework_3/recursion.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+function rec(nummer) {
+    if (nummer === 2) {
+        return true;
+    }
+    if (nummer % 2 === 0) {
```
Comment (markoffden) [inline homeworks/irina.vlasova_IrinaBondarenko/homework_3/recursion.js]: here it's more readable and logically appropriate to check `if (nummer > 2)`

File: homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js (hunk 2 above)
Comment (IrinaBondarenko) [reply homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js]: I`m so sorry, but I can`t use here .map( ) method. It returns array. ((

**Comment by IrinaBondarenko** [comment]
Changes done.Would you be so kind to check.Thanks in advance.

File: homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js (hunk 2 above)
Comment (markoffden) [reply homeworks/irina.vlasova_IrinaBondarenko/homework_3/function.js]: my typo, you got it right, it supposed to be `reduce()`

**Pull request merged**